lib/membrane_opus/encoder.ex

```diff
@@ -173,7 +171,29 @@ defmodule Membrane.Opus.Encoder do
   end
 
   defp set_current_pts(%{queue: <<>>} = state, input_pts) do
-    %{state | current_pts: input_pts}
+    if state.current_pts != nil and input_pts != nil and state.current_pts > input_pts do
+      diff = (state.current_pts - input_pts) |> Time.nanoseconds()
+      ms = diff |> Time.as_milliseconds(:round)
+
+      cond do
+        diff > Time.milliseconds(100) ->
+          Membrane.Logger.warning(
+            "Input buffer PTS is #{Time.pretty_duration(diff)} (#{ms} ms) smaller than expected value"
+          )
+
+        diff > Time.milliseconds(10) ->
+          Membrane.Logger.debug(
+            "Input buffer PTS is #{Time.pretty_duration(diff)} (#{ms} ms) smaller than expected value"
+          )
```

Comment:
Print diff only one (or in `ms`, or as result of `Time.pretty_duration`)
Add to the log expected value (`state.current_pts`) and value that occurs in the input buffer